### PR TITLE
feat: forward API logs to Sentinel

### PR DESCRIPTION
**Pull request by patheard**
# Summary 
Add the CloudWatch log group forwarder to send the API's logs to Sentinel.

# Related
- #265

### CODE REVIEW

File: terragrunt/aws/api/sentinel.tf [hunk 1]
```diff
@@ -0,0 +1,17 @@
+locals {
+  api_log_group_arn            = "arn:aws:logs:${var.region}:${var.account_id}:log-group:/aws/lambda/${module.url_shortener_lambda.function_name}"
+  sentinel_forwarder_layer_arn = "arn:aws:lambda:ca-central-1:283582579564:layer:aws-sentinel-connector-layer:54"
```
Comment (cgye) [inline terragrunt/aws/api/sentinel.tf:3]: Is the account id the same for both staging and prod environments?

**Review by sylviamclaughlin — APPROVED** [review]
Looks good!

File: terragrunt/aws/api/sentinel.tf (hunk 1 above)
Comment (patheard) [reply terragrunt/aws/api/sentinel.tf:3]: Yup, for the layer ARN it is.  This is our SRE tools account where it gets published by this workflow step every time there's a change:
https://github.com/cds-snc/aws-sentinel-connector-layer/blob/b0e90a4dfafe0efafe305403478f60683bcdd9ea/.github/workflows/tf-apply.yml#L45-L48

**Review by cgye — COMMENTED** [review]
lgtm

**Pull request merged**